### DISCUSSION TITLE
How To Merge My Repository with Main Repository

### DISCUSSION THREAD

**Discussion by dawghouse73** [nmelhado/league-page#260] [Help!]
This is likely an issue with being unfamiliar with GitHub, but nonetheless, I'm stumped on how to merge my updates for my league and sync the updates from the main repository. 

- I reviewed the section on Updates (IV. Updates) on the main training_wheels page. The images shown do not match what I am in seeing in GitHub (possibly outdated images or I've done something wrong/unexpected). 

IV Updates Image
![image](https://github.com/nmelhado/league-page/assets/133917715/d70dc66b-6c12-4727-83f8-10cf15a0d267)

What I see
![image](https://github.com/nmelhado/league-page/assets/133917715/50eb232f-6e2a-41fd-ad0d-acad87ff01d4)

When I click the Sync Fork option I see this:
![image](https://github.com/nmelhado/league-page/assets/133917715/5f1fb27a-8a1f-4c42-a131-73f1288dae51)

I don't want to discard my previous updates made (yes, I know there's a lot - I was committing instead of previewing...I'm learning - I think). But to simply "Update Branch" - is this the new / equivalent to "Fetch and Merge" (as I don't see that option either)?

For what it's worth, I've been trying to customize my size, but staying within the primary confines of the default league setup. 

Short version: I don't know what to click to merge my sites to begin...and I'm unaware of what doing anything other than what the instructions from the IV. Updates section will tell me as I don't see the "Fetch Upstream" option.  Appreciate this is a user thing, but I am now struck with fear to update the site but clicking anything.

Please help.

**Comment by nmelhado** [comment]
Clicking "Sync fork" > "Update branch" should be the same as fetch upstream. If for whatever reason that doesn't work, you can always roll that back, so you won't lose your changes.

**Reply by kevmeist3r** [reply]
Yep, I ran into this issue last season also. It still appears our branches won't let us sync with your master when we select "Sync Fork" unless we discard all of the changes we've made to personalize our sites. So far the only option is to discard all of our commits, sync with the master, then make all the changes again.
